dialect_sqlite.go

```diff
@@ -229,15 +229,16 @@ func (m *sqlite) FizzTranslator() fizz.Translator {
 }
 
 func (m *sqlite) DumpSchema(w io.Writer) error {
-	cmd := exec.Command("sqlite3", m.Details().Database, ".schema")
+	cmd := exec.Command("sqlite3", m.Details().Database, ".schema --nosys")
```

Comment:
can we have a test case for this change?

Reply:
(reprod steps would be fine if it is not easy for testing!)

Reply:
added a simple test-case.